Automattic/wp-calypso
# Hide checklist when the migration is started

Closes #96417

## Proposed Changes
- Prevent the checklist rendering if the migration is "started"

![before](https://github.com/user-attachments/assets/c732d126-8f00-44f8-86bc-d94181668da8)
![after](https://github.com/user-attachments/assets/2f39f19a-fa7e-4297-ad8c-43f3e5c1158e)


## Why are these changes being made?
* When the migration is "started," the user can't make changes on the site, so showing the checklist doesn't make sense.

## Testing Instructions
* Go to /start
* Select any domain
* Click on "Import or migrate an existing site" without selecting any goal
* Choose the "DIFM"
* Stop on the credential step (Tell us about your WordPress site)
* Get the new site on the URL (E.g siteSlug=xxxx.wordpress.com)
* Go to /sites
* Search your new site
* Check there is "Migration Started" label
* Check there is no "Checklist" above the label. 


## Pre-merge Checklist
